src/utils/testFilters.ts

```diff
@@ -1,30 +1,65 @@
-import { getDate, getLatestDate } from "./formatDate";
+import {getDate, getTodayTests} from './formatDate';
 
 export const filterTestsExecution = (tests: any, _filters: any) => {
-    let filteredTestsExecution = tests;
+  let filteredTestsExecution = tests;
 
-    if ((_filters?.filter?.includes('error') || _filters?.filter?.includes('success') || _filters?.filter?.includes('pending'))) {
-        const filteredTests = filteredTestsExecution?.results?.filter((test: any) => _filters?.filter?.includes(test.status));
-        filteredTestsExecution = { ...filteredTestsExecution, results: filteredTests };
-    }
+  if (
```

Comment:
filtering should be entirely done server-side - since the response is paged you might not have any results to show if you filter on the client